bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/BindingConstants.java

```diff
@@ -107,4 +107,13 @@
     public static final String CONFIG_APIKEY = "apikey";
 
     public static final String UNIQUE_ID = "uid";
+
+    public static final String PROPERTY_CT_MIN = "ctmin";
+    public static final String PROPERTY_CT_MAX = "ctmax";
+
+    // CT value range according to ZCL Spec
```

Comment:
According to http://dresden-elektronik.github.io/deconz-rest-doc/lights/#getstate the range is far more limited to 153-500. We should use that for the range check.


| state.ct | Number (153..500) | Mired color temperature of the light. (2000K - 6500K)
-- | -- | --

Reply:
I disagree with the proposed value range. I already have a few lights outside of this range, The deconz documentation is wrong here. the range is only limited by the reported range by the light bulb in deconz.
See : [deconz rest plugin source code](https://github.com/dresden-elektronik/deconz-rest-plugin/blob/b7ac16f55294e5928993b2e19129d416f8e2e1a2/rest_lights.cpp#L670)

The Zigbee cluster secificaten defines a valid range for the ct values to be 1 - 65279 mired /  that would be 15 K - 1000000K
You can find that in the [spec ](https://zigbeealliance.org/wp-content/uploads/2019/12/07-5123-06-zigbee-cluster-library-specification.pdf)if you scroll to pdf page 327 is at the bottom. 
I don't see any reason to limit the value range other than to the zigbee standard allowed range.

Reply:
Ok. We can discuss if we set the range for The dynamic state description to that values if we don‘t receive useful values.